cgcnn2/cli/cgcnn_ft.py

```
@@ -458,13 +456,13 @@ def main():
             optimizer, mode="min", factor=factor, patience=int(lr_patience)
         )
         logging.info(
-            "* Learning rate adjustment is configured with a factor of {} and patience of {} epochs.".format(
+            "Learning rate adjustment is configured with a factor of {} and patience of {} epochs.".format(
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)